packages/functional-tests/tests/oauth/resetPassword.spec.ts

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+import { EmailHeader, EmailType } from '../../lib/email';
+
+let email;
+const password = 'passwordzxcv';
+const newPassword = 'Newpassword@';
+
+test.describe('oauth reset password', () => {
+  test.beforeEach(async ({ target, pages: { login } }) => {
+    test.slow();
+    email = login.createEmail();
+    await target.auth.signUp(email, password, {
+      lang: 'en',
+      preVerified: 'true',
```

Comment:
Ideally, would be nice to move away from `preVerified:true` here since production does not support it.

Reply:
removed.